src/utils/constants.ts

```diff
@@ -52,28 +43,36 @@ export const SKILLS: SkillName[] = [
   'farming',
   'runecraft',
   'hunter',
-  'construction'
-];
-export const CLUES: ClueType[] = [
+  'construction',
+  'sailing'
+] as const;
+export type SkillName = (typeof SKILLS)[number];
+export const CLUES = [
   'all',
   'beginner',
   'easy',
   'medium',
   'hard',
   'elite',
   'master'
-];
-export const BH_MODES: BHType[] = ['hunterV2', 'rogueV2', 'hunter', 'rogue'];
-export const GAMEMODES: Gamemode[] = [
+] as const;
+export type ClueType = (typeof CLUES)[number];
+export const BH_MODES = ['hunterV2', 'rogueV2', 'hunter', 'rogue'] as const;
+export type BHType = (typeof BH_MODES)[number];
+export const GAMEMODES = [
   'main',
   'ironman',
   'hardcore',
   'ultimate',
   'deadman',
   'seasonal',
-  'tournament'
-];
-export const BOSSES: Boss[] = [
+  'tournament',
+  'skiller',
+  'oneDefence',
+  'freshStart'
```

Comment:
Looks like these have been missing for some time and would have been causing the gamemode validations to fail.

Adding const assertions and deriving the types from these instead of the other way around should prevent anything like this from getting missed in the future.